rtv/page.py

```diff
@@ -369,11 +416,16 @@ def _add_cursor(self):
     def _remove_cursor(self):
         self._edit_cursor(curses.A_NORMAL)
 
-    def _move_cursor(self, direction):
+    def _move_cursor(self, direction, page_ud=False):
 
         self._remove_cursor()
 
-        valid, redraw = self.nav.move(direction, len(self._subwindows))
+        if page_ud:
+            valid, redraw = self.nav.move_page(direction,
+                                               len(self._subwindows))
```

Comment:
I agree with your first point. Check out `navigator.flip()`, you might be able to leverage that to change the screen orientation before paging. 

For the second issue, how about moving `max(1, len(subwindows) - 1)` instead of `len(subwindows)`. That way any partial post will still be visible. In the rare case where there isn't a partial post, you end up with bottom post moved to the top of the screen. I see this as a positive because it will help users reorient their position on the screen.